internal/config/config.go

```diff
@@ -1119,6 +1119,9 @@ type AzureSEVSNP struct {
 	// description: |
 	//   AMD Root Key certificate used to verify the SEV-SNP certificate chain.
 	AMDRootKey Certificate `json:"amdRootKey" yaml:"amdRootKey"`
+	// description: |
+	//   AMD Signing Key certificate used to verify the SEV-SNP VCEK / VLEK certificate.
+	AMDSigningKey Certificate `json:"amdSigningKey" yaml:"amdSigningKey" validate:"len=0"`
```

Comment:
```suggestion
	AMDSigningKey Certificate `json:"amdSigningKey,omitempty" yaml:"amdSigningKey,omitempty" validate:"len=0"`
```
Should be hidden when not in use